website/templates/search.mako

```diff
@@ -343,9 +314,11 @@
 </%def>
 
 <%def name="javascript_bottom()">
+    <% import website %>
```

Comment:
I wouldn't recommend importing all of `website` in a mako rendering context. The `routes.py` entry for this doesn't use a function, but instead just passes an empty dict to this template -- you can put `'shareUrl': settings.SHARE_URL` in there.